app/views/provider_interface/decisions/new_offer.html.erb

```diff
@@ -31,10 +31,10 @@
         <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">Further conditions (optional)</legend>
         <p class="govuk-body">Outline any further conditions (for example, studying a subject knowledge enhancement course) and give deadlines for completing them.</p>
 
-        <%= f.govuk_text_area :further_conditions, label: { text: 'First condition', size: 's' }, id: 'first_condition', rows: 3, multiple: true %>
-        <%= f.govuk_text_area :further_conditions, label: { text: 'Second condition', size: 's' }, id: 'second_condition', rows: 3, multiple: true %>
-        <%= f.govuk_text_area :further_conditions, label: { text: 'Third condition', size: 's' }, id: 'third_condition', rows: 3, multiple: true %>
-        <%= f.govuk_text_area :further_conditions, label: { text: 'Further condition', size: 's' }, id: 'further_condition', rows: 3, multiple: true %>
+        <%= f.govuk_text_area :further_conditions0, label: { text: 'First condition', size: 's' }, rows: 3, max_chars: 255, threshold: 75 %>
```

Comment:
I don't think we should be adding a countdown unless we know one is useful from research.

Is 255 characters just the default setting or is it limited to this for a reason?

I think we should trust providers to say what they need to say. That might mean we up the limit to avoid constrainting providers too much and avoid getting the error at all.

Reply:
I'm not sure where the limit comes from. Technically there is nothing stopping us storing arbitrary length condition text values.